<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>160</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Money input</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ascend_10">
      <property name="text">
       <string>10원</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ascend_50">
      <property name="text">
       <string>50원</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ascend_100">
      <property name="text">
       <string>100원</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ascend_500">
      <property name="text">
       <string>500원</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>160</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="descend_100">
      <property name="text">
       <string>Coffee(100)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="descend_150">
      <property name="text">
       <string>Tea(150)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="descend_200">
      <property name="text">
       <string>Milk(200)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>158</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reset_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
